cron/cron_task.go

```diff
@@ -35,21 +35,43 @@ func (s *CronService) StartCronService() {
 			}
 		}()
 
-		runValue := true
-		skip := 0
-		for runValue {
-			total, err := UpdateUnknownTxsByPage(skip, 20)
-			if err != nil {
-				logger.Error("GetUnknownTxsByPage have error", logger.String("err", err.Error()))
+		fn_update_unknown_txs := func() {
+			runValue := true
+			skip := 0
+			for runValue {
+				total, err := UpdateUnknownTxsByPage(skip, 20)
+				if err != nil {
+					logger.Error("getCommonTx have error", logger.String("err", err.Error()))
+				}
+				if total < 20 {
+					runValue = false
+					logger.Info("Finish UpdateUnknownTxsByPage.", logger.Int("total", total))
+				} else {
+					skip = skip + total
+					logger.Info("Continue UpdateUnknownTxsByPage", logger.Int("skip", skip))
+				}
 			}
-			if total < 20 {
-				runValue = false
-				logger.Info("Finish UpdateUnknownTxsByPage.", logger.Int("total", total))
-			} else {
-				skip = skip + total
-				logger.Info("Continue UpdateUnknownTxsByPage", logger.Int("skip", skip))
+		}
+		fn_update_emptytype_txs := func() {
```

Comment:
combine `fn_update_emptytype_txs` and `fn_update_unknown_txs`,  use one function to update txs which status is unknown or txType is empty